haystack/nodes/file_converter/csv.py

```diff
@@ -29,11 +29,23 @@ def convert(
         id_hash_keys: Optional[List[str]] = None,
     ) -> List[Document]:
         """
-        Load CVS file and convert it to documents.
-
-        :param file_path: Path to a CSV file containing two columns.
-            The first will be interpreted as a question, the second as content.
-        :returns: List of document, 1 document per line in the CSV.
+                Load a CVS file containing question-answer pairs and convert it to Documents.
+
+                :param file_path: Path to the CSV file you want to convert. The file must have two columns called 'question' and 'answer'.
+                    The first will be interpreted as a question, the second as content.
+                :param meta: A dictionary of metadata key-value pairs that you want to append to the returned document. It's optional.
+                :param remove_numeric_tables: This option uses heuristics to remove numeric rows from the tables.
```

Comment:
This option is not used by this converter, so it has no effect for this node. It's still been added to follow the function signature expected for the `convert` function.